Source/Fuse.Common/UpdateManager.uno

```diff
@@ -82,8 +81,18 @@ namespace Fuse
 		public List<UpdateListener> Onces = new List<UpdateListener>();
 		public List<UpdateListener> OncesPending = new List<UpdateListener>();
 		
-		public List<UpdateAction> PhaseDeferredActions = new List<UpdateAction>();
-		public int PhaseDeferredActionsAt = -1;
+		public Dictionary<int, Queue<UpdateAction>> PhaseDeferredActions = new Dictionary<int, Queue<UpdateAction>>();
```

Comment:
I wonder if we shoudln't change the values in the `LayoutPriority` enum then to be from 0...N instead, making it clear that you shouldn't do things like +/-1 on the priority (which it looks like you can do now based on the enum values).